### PR TITLE
10.0.x

### PR DESCRIPTION
**Pull request by mhevery**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,40 @@
-<!--
 <a name="10.0.0"></a>
-# [10.0.0](https://github.com/angular/angular/compare/10.0.0-next.0...10.0.0) (2020-06-24)
-
+# [10.0.0](https://github.com/angular/angular/compare/10.0.0-rc.6...10.0.0) (2020-06-24)
 
 ### Release Highlights & Update instructions
 To learn about the release highlights and our CLI-powered automated update workflow for your projects please check out the [v10 release announcement](https://blog.angular.io/TODO).
```
Comment (mhevery) [inline CHANGELOG.md:5]:
```suggestion
To learn about the release highlights and our CLI-powered automated update workflow for your projects please check out the [v10 release announcement](https://blog.angular.io/TODO).
```

Still waiting on @StephenFluin for the URL.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
 
-#### Dependency updates
-@angular/compiler-cli now requires:
-- TypeScript 3.9
-
-TODO: concat all the 10.0.0-next.* & -rc.* release notes, and group them by area/package just like in https://github.com/angular/angular/blob/master/CHANGELOG.md#900-2020-02-06 but consider listing features before bugfixes because people care about feature in major releases more than about bug fixes.
--->
+### Features
 
+* **bazel:** expose explicit mapping from closure to devmode files ([#36262](https://github.com/angular/angular/issues/36262)) ([ba796bb](https://github.com/angular/angular/commit/ba796bb))
```
Comment (IgorMinar) [inline CHANGELOG.md:10]: can you please sort the entries here by scope (e.g. all "bazel" stuff together, and all "core" stuff together).

**Review by IgorMinar — APPROVED** [review]
LGTM otherwise! Thank you!

Reviewed-for: global-approvers

File: CHANGELOG.md (hunk 2 above)
Comment (mhevery) [reply CHANGELOG.md:10]: done

**Pull request merged**